tests/python_raise/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -0,0 +1,75 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`raise.py 1`] = `
+raise
+raise ValueError
+raise ValueError("error")
+
+raise NotImplementedError('example.')
+
+raise forms.ValidationError(
+    self.error_messages['invalid_login'],
+    code='invalid_login',
+    params={
+        'username': self.username_field.verbose_name
+    }
+)
+
+raise SomeErrorWithAReallyReallyVeryVeryExtremelyExtensivelyLongNameSoItBreaks()
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+raise
+raise ValueError
+raise ValueError("error")
+
+raise NotImplementedError("example.")
+
+raise (
+    forms.ValidationError(self.error_messages[
+        "invalid_login"
+    ], code="invalid_login", params={
+        "username": self.username_field.verbose_name
+    })
+)
```

Comment:
Yeah, I agree tis is a bit funky, but I think it's the best we can do right now. The functionality in #52 would likely help here.